packages/database-version-control/src/migrations/2023.04.19T14.08.58.create-application-status-table.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS bng.application_status
 (
 	application_status_id UUID CONSTRAINT pk_application_status_application_status_id PRIMARY KEY DEFAULT uuid_generate_v4(),
-	application_reference VARCHAR(20),
+	application_reference VARCHAR(22),
```

Comment:
This needs a new migration to alter the column length as the migration has been merged to master and deployed to cloud environments.